src/data/home/Values.js

```diff
@@ -1,25 +1,24 @@
 import React from "react";
-import CommunityIcon from "@/public/assets/community.svg";
-import GradCap from "@/public/assets/gradcap.svg";
-import Temple from "@/public/assets/temple.svg";
-import Image from "next/image";
+import { MdTempleBuddhist } from "react-icons/md";
```

Comment:
import { GiIndianPalace } from "react-icons/gi";

try using this instead of the temple